### ISSUE TITLE
Buchungskalender für Benutzer wird nicht angezeigt

### ISSUE THREAD

**Issue by mimosel** [tiloman/simren-issues#85]
Der Buchungskalender für Benutzer wird nicht angezeigt, auch wenn die Option aktiviert ist.

(siehe https://github.com/tiloman/simren-issues/issues/72 )

**Comment by tiloman** [comment]
<img width="1291" alt="image" src="https://github.com/tiloman/simren-issues/assets/47814142/5f6d06d4-f64d-438e-86fe-1093aef51cbc">

Kalender ist sichtbar in der jeweiligen Ressource.

**Comment by mimosel** [comment]
Aber nur, wenn ich auf "Infos" zu einer Ressource gehe und nicht, wenn ich auch "Buchen" bei der Ressource gehe.
Warum gibt es eigentlich Infos und Buchen? Die sind doch ziemlich redundant... und wieso wird der Kalender nicht auch bei "Buchen" angezeigt?

**Comment by tiloman** [comment]
Ja, bei den Infos gibt es allgemeine Infos zur Ressource (Beschreibung, wie lange sie verliehen werden kann, etc.). Wenn man sich dann dafür entscheidet, klickt man auf `Buchen` und gelangt konkret zum Buchungsformular. 
Wenn man aus der Ressourcenansicht auf `Buchen` klickt (jemand weiß schon konkret was er möchte und braucht keine Infos mehr), kommt man direkt auf die Buchungsmaske.

> und wieso wird der Kalender nicht auch bei "Buchen" angezeigt?
Dort gibt es einen Datepicker mit dem man dann das gewünschte Datum auswählt da die Buchung von Ressourcen in der Regel mehrtägig ist. In der Buchungsmaske für Räume wählt man den Zeitraum in einem Kalender aus.

**Comment by mimosel** [comment]
Aber wäre es nicht voll praktisch, wenn man den Kalender auch im Buchungsformular sehen würde? Ich mein, da braucht man die Infos aus dem Kalender ja und der Datepicker ist von der Funktionalität her ja deutlich schlechter... 
Ich für meinen Teil würde die Infos aus "Infos" wirklich komplett in die Buchungsmaske packen und die "Info"-Seite abschaffen...

**Comment by tiloman** [comment]
Es ist geplant den Datepicker noch weiter auszubauen (aktuell werden dort nur Zeiten außerhalb der Öffnungszeiten ausgegraut und Sperrzeiten). Ich werde auch die Verfügbarkeit in Zukunft prüfen und die Daten im Vorfeld gar nicht anzeigen an denen Ressourcen ausgebucht sind. 

Ich könnte den Kalender per Popup einblendbar machen. Das gilt dann aber nur für die Hauptressource. Wenn mehrere Ressourcen kombiniert werden möchte man ja auch mehrere Kalender und dann wird es unübersichtlich ... Ich mache mir aber Gedanken dazu. 

> Ich für meinen Teil würde die Infos aus "Infos" wirklich komplett in die Buchungsmaske packen und die "Info"-Seite abschaffen...

Wenn die Beschreibung zum Beispiel sehr lang ist, wird die Buchungsseite sehr schnell unübersichtlich und die würde ich so einfach wie möglich halten. Auch hier gibt es wieder das `Problem` dass mehrere Ressourcen kombiniert werden können.

**Comment by OneOfTwelve** [comment]
Für uns wäre es besser wenn die Infos eingeklappt bleiben.
Wir haben in den Infos eine Beschreibung des Gerätes so wie zum Beispiel das Zubehör welches wir mit dem Gerät mitgeben. Gerade am Handy würde ein dauerhaftes anzeigen schnell in unnötig viel gescrolle ausarten, wenn man schon weiß was man will.

**Comment by mimosel** [comment]
Okay, okay... Ihr habt ja Recht.

**Comment by mimosel** [comment]
> Es ist geplant den Datepicker noch weiter auszubauen (aktuell werden dort nur Zeiten außerhalb der Öffnungszeiten ausgegraut und Sperrzeiten). Ich werde auch die Verfügbarkeit in Zukunft prüfen und die Daten im Vorfeld gar nicht anzeigen an denen Ressourcen ausgebucht sind.

Gibt es hierfür einen Zeitplan?

**Comment by tiloman** [comment]
Nein, nicht direkt. Der Change greift jedenfalls recht tief in das System ein und ist daher nicht ganz so schnell umsetzbar.

**Comment by tiloman** [comment]
Mittlerweile werden Tage an denen Ressourcen komplett ausgebucht sind auch ausgegraut und sind nicht auswählbar
